lib/usdUfe/python/wrapUtils.cpp

```diff
@@ -104,11 +101,12 @@ void wrapUtils()
     def("usdPathToUfePathSegment",
         _usdPathToUfePathSegment,
         (arg("usdPath"), arg("instanceIndex") = PXR_NS::UsdImagingDelegate::ALL_INSTANCES));
-    def("uniqueChildName", _uniqueChildName);
+    def("uniqueName", _uniqueName);
+    def("uniqueChildName", UsdUfe::uniqueChildName);
```

Comment:
For some of these we don't need the extra _ helper defined here. Can just call the UsdUfe function directly.